common/client/node_lifecycle_test.go

```diff
@@ -471,21 +474,24 @@ func TestUnit_NodeLifecycle_outOfSyncLoop(t *testing.T) {
 			return node.State() == nodeStateUnreachable
 		})
 	})
-	t.Run("if fail to get chainID, transitions to invalidChainID", func(t *testing.T) {
+	t.Run("if fail to get chainID, transitions to unreachable", func(t *testing.T) {
```

Comment:
Why this change in logic?

Reply:
This way it's possible to unify reconnection behaviour and reuse it in multiple state transitions and makes it easier to add additional verification steps. 

Before the change, error handling on chainID verification error for `outOfSync` loop looked like this
```
if err := n.verify(n.nodeCtx); err != nil {
		lggr.Errorw(fmt.Sprintf("Failed to verify out-of-sync RPC node: %v", err), "err", err)
		n.declareInvalidChainID()
		return
}
```

It was not consistent with the rest of the loops and produced state transition that seems redundant `outOfSync` -> `invalidChainID` -> `unreachable` instead of `outOfSync` -> `unreachable`